server/parsec/events.py

```diff
@@ -277,6 +294,12 @@ class EventShamirRecoveryCertificate(BaseModel, ClientBroadcastableEvent):
 
     @override
     def is_event_for_client(self, client: RegisteredClient) -> bool:
+        # Note, unlike for vlob, the certificate author still get a event about it.
+        #
+        # This is because certificates are always integrated in the client by first
+        # polling them from the server (in order to ensure no concurrently created
+        # certificate we are not yet aware of is older than the certificate we've
+        # just created).
```

Comment:
```suggestion
        # Note, unlike for vlob, the certificate author still gets notified about the event.
        #
	    # This is because certificates are always integrated into the client by first 
	    # polling them from the server (in order to ensure no concurrently-created 
	    # certificate, of which the client may not be aware yet, is older than the 
	    # certificate it has just created).
```